config/config-package.py

```diff
@@ -265,6 +265,7 @@ def pre_commit_config(self):
                 'codespell_extra_lines',
                 'flake8_extra_lines',
                 'extra_lines',
+                'exclude_files'
```

Comment:
Almost there! 💯 

Notice that the other options are named `TOOL_extra_lines`, to make it simple and obvious what you are configuring.

In this PR I would name the option `i18ndude_extra_lines` so not only exclude configuration options can be given, but literally anything that has to do with configuring `i18ndude` within `pre-commit`